sources.go

```diff
@@ -83,6 +83,31 @@ func ListFiles(path string) *Pipe {
 	return Slice(fileNames)
 }
 
+// FindFiles creates a pipe containing the files inside a directory and all its subdirectories
```

Comment:
Also, you'll notice that comments are wrapped to about 80 columns (I use the Rewrap extension in VS Code for this, which is very handy).